src/__tests__/stateFromElement-test.js

```diff
@@ -9,23 +9,24 @@ import {
 import {convertToRaw} from 'draft-js';
 import {jsdom} from 'jsdom';
 import fs from 'fs';
+import {EOL} from 'os';
 import {join} from 'path';
 
 let document = jsdom('<!doctype html><html><body></body></html>');
 
 // This separates the test cases in `data/test-cases.txt`.
-const SEP = '\n\n#';
+const SEP = `${EOL}${EOL}#`;
```

Comment:
@sstur Most windows git clients are configured to checkout files with OS dependent line endings. https://help.github.com/articles/dealing-with-line-endings/#global-settings-for-line-endingsI need to change this to get the tests running on my windows machine.